src/jobs/deploy.yml

```diff
@@ -51,4 +51,8 @@ steps:
   - run:
       name: Deploy
       working_directory: dist
-      command: gcloud app deploy --version << parameters.version >> --verbosity=info ./app.yml
+      command: |
+        # --version/-v: May only contain lowercase letters, digits, and hyphens. Must begin and end with a letter or digit. Must not exceed 63 characters.
+        VERSION=$(python3 -c "import re, unicodedata; print(re.sub(r'(^-+|-+$)', '', re.sub(r'[^a-z0-9-]+', '-', ''.join(c for c in unicodedata.normalize('NFD', '<< parameters.version >>') if unicodedata.category(c) != 'Mn').lower())[:63-len('-dot-<< parameters.training-name >>-dot-zen-formations')]))")
```

Comment:
Could we maybe extract this to a file and properly format it? I would be easier to understand and maintain.

Reply:
Sadly it's not possible in an orb, unless it's the full script then included as `command: << include(scripts/script.sh) >>`

I changed to use a multiline python inline script.

Reply:
I added special case for `main` and `master` to not alter them + a minimum size of 4.

Reply:
OK, it already a lot better when formatted.

Reply:
Yes, way better! Thanks for the feedback.